tests/ballerina-spec-conformance-tests/S05_values_types_variables/tests/value_belongs_to_type.bal

```diff
@@ -23,9 +23,9 @@ const EXPECTED_VALUE_NOT_TO_BELONG_TO_TYPE_FAILURE_MESSAGE = "expected value to
 // look like the type no matter how the value is mutated
 @test:Config {}
 function testBelongsToWithArrays() {
-    (int|boolean)[] a1 = [true, false, false];
+    (int|boolean)[3] a1 = [true, false, false];
     any a2 = a1;
-    if !(a2 is anydata[]) || !(a2 is (int|boolean)[]) {
+    if !(a2 is anydata[3]) || !(a2 is (int|boolean)[3]) {
```

Comment:
`a2 is anydata[]` should also work (without the length). Seems like an issue with the `is` check.

Shall we create an issue if there isn't one already and add a comment here?

```suggestion
    // <GITHUB_ISSUE> 
    // if !(a2 is anydata[]) || !(a2 is (int|boolean)[3]) {
```